### PR TITLE
Improved error handling on WebException Sniper

### PR DESCRIPTION
**Pull request by mvegter197**

### CODE REVIEW

**Comment by DurtyFree** [comment]
You should still catch the other exceptions, just in case.

File: PoGo.PokeMobBot.Logic/Tasks/SnipePokemonTask.cs [hunk 1]
```diff
@@ -404,6 +404,10 @@ private static ScanResult SnipeScanForPokemon(ISession session, Location locatio
                 }
                 else
                 {
+                    session.EventDispatcher.Send(new ErrorEvent
```
Comment (DurtyFree) [inline PoGo.PokeMobBot.Logic/Tasks/SnipePokemonTask.cs]: You won't get this far if theres an exception in the code, not web related.
Just add a catch (Exception ex) under the ending braces of the WebException catch

**Comment by DurtyFree** [comment]
See my comment @mvegter197

**Comment by mvegter197** [comment]
@DurtyFree Did as you requested and added extra error handling as skiplagged also provides there own errors.

**Comment by DurtyFree** [comment]
Thanks!:)

**Pull request merged**